bigquery_datatransfer/nox.py

```diff
@@ -65,3 +95,18 @@ def system(session, py):
 
     # Run py.test against the system tests.
     session.run('py.test', '--quiet', 'tests/system/')
+
+
+@nox.session
+def cover(session):
```

Comment:
@lukesneeringer Is there a reason `cover` commands aren't generated? Is this intentional or an oversight or some 3rd option?

Reply:
Mostly oversight.